crates/valence_command_derive/README.md

```diff
@@ -0,0 +1,104 @@
+Simplify the creation of Valence commands with a derive macro.
+
+## Usage
+
+```rust
+#[derive(Command, Debug, Clone)]
+#[paths("teleport", "tp")]
+#[scopes("valence:command:teleport")]
+enum Teleport {
```

Comment:
I still think that `paths` is not a great name for this. What if we renamed it to `aliases`, `command` or `literal`?

Reply:
aliases doesn't reflect the fact it is a path, command is a bit to similar to the macro name for me but it could work and literal is a bit too out there.

Reply:
Yeah, I suppose it would be better to just elaborate more in the docs.